EtlPipelineBlobTrigger/tests/test_accreditations.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+import unittest
+import xml.etree.ElementTree as ET
+
+import xmltodict
+
+from course_accreditations import get_accreditations
+from accreditations import Accreditations
+
+
+def get_string(filename):
```

Comment:
I've written a shared get_string call that can be used here -  `from testing_utils import get_string`